atlasdb-client/src/main/java/com/palantir/atlasdb/logging/LoggingArgs.java

```diff
@@ -82,13 +86,29 @@ public static SafeAndUnsafeTableReferences tableRefs(Collection<TableReference>
                 .build();
     }
 
+    public static Iterable<TableReference> tableRefValues(Collection<TableReference> tables) {
+        //noinspection StaticPseudoFunctionalStyleMethod - Use lazy iterator.
+        return Iterables.transform(tables, LoggingArgs::tableRefValue);
+    }
+
     /**
      * Returns a safe or unsafe arg corresponding to the supplied table reference, with name "tableRef".
      */
     public static Arg<String> tableRef(TableReference tableReference) {
         return tableRef("tableRef", tableReference);
     }
 
+    /**
+     * If safe, returns the table name. If unsafe, returns {table}.
+     */
+    public static TableReference tableRefValue(TableReference tableReference) {
```

Comment:
this method name is odd, and the return value is not an `Arg` as the class would suggest

Reply:
Renamed them to `safeTableOrPlaceholder`, but still doesn't seem good. I'm open to suggestions at this point.